weave/trace/autopatch.py

```diff
@@ -89,6 +90,9 @@ def autopatch(settings: Optional[AutopatchSettings] = None) -> None:
     from weave.integrations.notdiamond.tracing import get_notdiamond_patcher
     from weave.integrations.openai.openai_sdk import get_openai_patcher
     from weave.integrations.openai_agents.openai_agents import get_openai_agents_patcher
+    from weave.integrations.openai_realtime.openai_realtime_sdk import (
+        get_websocket_patcher,
```

Comment:
this will change soon, but for now I recommend calling this specifically the `openai_realtime_websocket_patcher`

Reply:
Just to clarify:

You mean to change `get_websocket_patcher` -> `openai_realtime_websocket_patcher` correct?